### PR TITLE
687 Add "Hiring Guide" page under Learning Resources menu item with l…

### PR DESCRIPTION
**Pull request by kristyj**
Closes issue #687 - adds the list of interview question suggestions brainstormed at WTD Prague 2018 to the website.
Note: Leaves room for expansion of this new section for other questions discussed on the career-advice channel.

### CODE REVIEW

**Comment by plaindocs** [comment]
This is great! 

I'm not a 100% sure about creating a new Guide rather than just adding a Hiring heading to the main Guide, what do you think?

When we do merge it in, I'd go as far as suggesting pinging the newsletter folks and asking for a mention.

**Review by plaindocs — APPROVED** [review]

**Comment by ericholscher** [comment]
> I'm not a 100% sure about creating a new Guide rather than just adding a Hiring heading to the main Guide, what do you think?

Yea, I'd just include it under the main guide, until we have enough stuff to justify breaking it out.

**Comment by ericholscher** [comment]
Actually, I'm having second thoughts. I do think career/hiring is a different topic than the "doc guide" that we have. I do think it maybe makes sense to break out as a different guide. 

I wonder if having it as a different section will also make it easier to contribute to, and build out as a contained resource.

**Comment by ericholscher** [comment]
Also, thanks so much for this PR @kristyj! It's 💯

**Comment by plaindocs** [comment]
I can see both sides of this. "Documentation Culture" that is already included in the Guide should probably live by the same logic? Kinda depends if the Guide is about writing Docs or about Doc related stuff... 

> This guide exists to provide both novice and expert writers a best practice handbook for building, structuring, and writing software documentation.

File: docs/hiring-guide/index.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
+============
+Hiring Guide
```
Comment (mjang) [inline docs/hiring-guide/index.rst]: Great contribution!

When I read "Hiring Guide", I think this advice is limited to hiring managers and HR, so I'm trying to think of a different title. (and that's my only concern with the PR)

Maybe "Guide to Hiring and Getting Hired"?

Comment (mjang) [reply docs/hiring-guide/index.rst]: Or maybe "Hiring and Interviewing" (brainstorming -- I have an open mind on this)

Comment (kristyj) [reply docs/hiring-guide/index.rst]: I was definitely wrestling with the question of the name too @mjang. I like your suggestion of "Guide to Hiring and Getting Hired" but perhaps it's a little long for the sidebar - how would you feel about keeping it the same in the sidebar but using the new title on the page itself?

**Review by mjang — APPROVED** [review]
wfm, thanks!

**Comment by kristyj** [comment]
Thanks all for your reviews so far! @ericholscher I did feel that the "Documentation guide" section gives a strong first impression that it a resource only for writing advice and tools, hence wanting to split this topic into a separate section. The audience may be hiring managers who do not know much about docs and will need a visible location to access this information. 

> I wonder if having it as a different section will also make it easier to contribute to, and build out as a contained resource.

I recognise that it seems a bit bare now, but it would be great if we could encourage contributors! There seems to be a lot of activity on slack that could be added here for posterity.

**Comment by ericholscher** [comment]
I think this looks great for now, and agree we can iterate on it over time. I'm gonna merge it to get the info out there, and we can work on it more over time with it live. 🎉 

Thanks for the PR/input folks!

**Pull request merged**